### DISCUSSION TITLE
deaths on the same tick

### DISCUSSION THREAD

**Discussion by cfpperche** [saul/demofile#285] [Questions] [answered]
Hi, a little doubt

can two or more players die on the same tick?

thanks

**Comment by saul** [comment, marked as answer]
Do you mean can they trade kills? e.g. player A kills player B on the same tick player B kills player A?

**Reply by cfpperche** [reply]
No, I know it doesn't happen in CS: GO, it would be like Player A kills Player B and Player C kills Player D all on the same tick

**Reply by saul** [reply]
Yes - that is definitely possible.

**Reply by cfpperche** [reply]
Ok! Thank you
